app/controllers/foreman_xen/cache_controller.rb

```diff
@@ -0,0 +1,34 @@
+module ForemanXen
+  class CacheController < ::ApplicationController
+    helper :all
+    skip_before_action :verify_authenticity_token
```

Comment:
its not, the token is also available as JS variable (https://github.com/theforeman/foreman/blob/develop/app/views/layouts/base.html.erb#L27)